command/version/testdata/pretty.golden

```diff
@@ -0,0 +1,3 @@
+Consul 1.99.3-beta1
+Revision 5e5dbedd47a5f875b60e241c5555a9caab595246
```

Comment:
I don't know if this is an issue but there may be a tool/check in people's pipelines that parses the protocol from the 2nd line or `tail -n 1` or something where this may break.  In which case we would suggest them use the json formatter from now on.

Reply:
Should this go into v1.9 then?

Reply:
Hmm, thinking about it I am not sure whether we should consider this a breaking change. We provide no guarantees about the output.